### DISCUSSION TITLE
Specifying shape by name

### DISCUSSION THREAD

**Discussion by MaxGasta** [OrchardCMS/OrchardCore#7170] [Q&A] [answered]
Hello,

with liquid  works fine
{{ Model.Content | shape_build_display: "Summary" | shape_render }}

**How about in Razor?**

from Orchardcore Reference I'm trying 
@Model.Content.**Named**("Summary");

but I get an error:  "Named" method NOT found
Am I missing a using?
Any help?

Thanks for your attention

**Comment by ns8482e** [comment, marked as answer]
Inject `IContentItemDisplayManager` and call `BuildDisplayAsync` with display type "Summary"

**Reply by MaxGasta** [reply]
thanks a lot!
